src/Compilers/VisualBasic/Portable/Operations/VisualBasicOperationFactory.vb

```diff
@@ -869,14 +845,40 @@ Namespace Microsoft.CodeAnalysis.Semantics
 
         Private Function CreateBoundSelectStatementOperation(boundSelectStatement As BoundSelectStatement) As ISwitchStatement
             Dim value As Lazy(Of IOperation) = New Lazy(Of IOperation)(Function() Create(boundSelectStatement.ExpressionStatement.Expression))
-            Dim cases As Lazy(Of ImmutableArray(Of ISwitchCase)) = New Lazy(Of ImmutableArray(Of ISwitchCase))(Function() GetSwitchStatementCases(boundSelectStatement.CaseBlocks))
+            Dim cases As Lazy(Of ImmutableArray(Of ISwitchCase)) = New Lazy(Of ImmutableArray(Of ISwitchCase))(Function() boundSelectStatement.CaseBlocks.SelectAsArray(Function(n) DirectCast(Create(n), ISwitchCase)))
             Dim syntax As SyntaxNode = boundSelectStatement.Syntax
             Dim type As ITypeSymbol = Nothing
             Dim constantValue As [Optional](Of Object) = New [Optional](Of Object)()
             Dim isImplicit As Boolean = boundSelectStatement.WasCompilerGenerated
             Return New LazySwitchStatement(value, cases, _semanticModel, syntax, type, constantValue, isImplicit)
         End Function
 
+        Private Function CreateBoundCaseBlockOperation(boundCaseBlock As BoundCaseBlock) As ISwitchCase
```

Comment:
>Private Function CreateBoundCaseBlockOperation(boundCaseBlock As BoundCaseBlock) As ISwitchCase [](start = 8, length = 95)

Is this change a part of an unrelated commit?

Reply:
I see, it looks like the code was moved from another file.

---
In reply to: [140051547](https://github.com/dotnet/roslyn/pull/21857#discussion_r140051547) [](ancestors = 140051547)

Reply:
no it was there before. case block wasn't go through the bound node -> ioperation map before, so I fixed it.

Reply:
ah, ya, it was called directly from case statement or something, so I made it to go through Create(...) but implementation was already there. I just connected it to Create.